src/generic/grid.cpp

```diff
@@ -2625,6 +2625,9 @@ void wxGridWindow::OnMouseWheel( wxMouseEvent& event )
 {
     if (!m_owner->ProcessWindowEvent( event ))
         event.Skip();
+
+    // To redraw grid after scroll. If not called - some lines are not updated in grids where cell is combined from multiple items.
+    m_owner->Refresh();
```

Comment:
Why do we only need to do this when scrolling using the wheel? Couldn't the same thing happen when dragging the scrollbar thumb around with the mouse?

Reply:
In my experiments I did not noticed any issues when scrollbar was used. Refresh was only required after mouse scroll.